README.md

```diff
@@ -20,6 +20,7 @@ addon | version | summary
 [queue_job](queue_job/) | 12.0.1.0.0 | Job Queue
 [queue_job_cron](queue_job_cron/) | 12.0.1.0.1 | Scheduled Actions as Queue Jobs
 [test_queue_job](test_queue_job/) | 12.0.1.0.0 | Queue Job Tests
+[base_export_async](base_export_async/) | 12.0.1.0.0 | Export (CSV and XLS) files in the background
```

Comment:
note this file is auto generated

Reply:
I guess this is linked to your previous comment about the autogenerated readme.

Reply:
yes and no, actually it's this specific section of README.md that is generated, so it's not mandatory to update it manualy... but it does no harm.